staker-rewards/test/helpers/blockchain.js

```diff
@@ -0,0 +1,37 @@
+import testBlockchain from "../data/test-blockchain.json"
+
+const getStateAtBlock = (contractAddress, blockNumber) => {
+  const block = testBlockchain.find(
+    (block) => block.blockNumber === blockNumber
+  )
+
+  return (
+    block &&
+    block.contracts.find((contract) => contract.address === contractAddress)
+  )
+}
+
+// inputCheck is expected to be a function executed to match input parameters
```

Comment:
A reference to `test-blockchain.json` here would help to understand what the input is matched against.